vktrace/vktrace_layer/vktrace_lib_trace.cpp

```diff
@@ -5360,7 +5360,10 @@ VKTRACER_EXPORT VKAPI_ATTR PFN_vkVoidFunction VKAPI_CALL __HOOKED_vkGetInstanceP
 }
 
 static const VkLayerProperties layerProps = {
-    "VK_LAYER_LUNARG_vktrace", VK_MAKE_VERSION(1, 0, VK_HEADER_VERSION), 1, "LunarG tracing layer",
+    "VK_LAYER_LUNARG_vktrace",
+    VK_MAKE_VERSION(1, 2, VK_HEADER_VERSION),
```

Comment:
did we actually make `vktrace` changes for 1.2.131?  I thought adding new stuff to the `vktrace` layers was difficult, and that we were actually a few header versions back already...

Reply:
Yes, we did. See commit 6c9cc0f956af4e960d0a9d6b9e1bb9933cd5aa90